spyder/plugins/editor/confpage.py

```diff
@@ -48,6 +48,7 @@ def setup_page(self):
 
         # ---- Display tab
         showtabbar_box = newcb(_("Show tab bar"), 'show_tab_bar')
+        show_filename_box = newcb(_("Show file name"), 'show_filename_toolbar')
```

Comment:
```suggestion
        show_filename_box = newcb(
            _("Show file name on top of tab bar"), 'show_filename_toolbar')
        )
```

I think my previous suggestion was not clear enough, i.e. we need a more descriptive name here.